src/cljs/main/broadfcui/page/workspace/summary/acl_editor.cljs

```diff
@@ -10,11 +10,11 @@
     [broadfcui.utils :as utils]
     ))
 
-
+; Note that render-acl-content below requires that these values be in descending order
 (def ^:private access-levels ["OWNER" "WRITER" "READER" "NO ACCESS"])
```

Comment:
I also just noticed another copy of this: `access-levels-ascending` from common.cljs. Can that be used instead of having this copy here? (`access-levels-ascending` does include `PROJECT_OWNER`).

I actually suspect that there might actually be a difference between these two lists that I don't understand, so describing why they are two separate lists would be acceptable as well.